build-product-image/action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   build-cache-password:
     description: Build cache password
     required: true
+  bake-config-file:
+    description: Path to the bake config file, defaults to `./conf.py`
+    default: ./conf.py
```

Comment:
I believe this should have an associated entry in the doc

Reply:
Ah yes, true. Missed that.

Reply:
Done in f637206